app/controllers/internal_api/v1/pillars_controller.rb

```diff
@@ -129,5 +131,25 @@ def openstack_cloud_contents
       }
     }
   end
+
+  def kubelet_contents
+    reservations ={}
+    KubeletComputeResourcesReservation.all.each do |r|
+      reservation = {
```

Comment:
we don't necessarily need to cache this in a variable, as it's just used once in the loop